lms/static/moment_requirejs.js

```diff
@@ -0,0 +1,15 @@
+(function(requirejs) {
```

Comment:
I don't understand this as well and this shouldn't be necessary. But as fat as I know this file is here for historical reasons, some issues with moment.js. I think @muhammad-ammar can better explain this.